utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-deployments.yaml

```diff
@@ -105,9 +105,11 @@ spec:
           - name: cloudharness-allvalues
             mountPath: /opt/cloudharness/resources
             readOnly: true
+          {{- if has  "accounts" .app.harness.dependencies.hard }}
```

Comment:
@filippomc did you add accounts as a hard dependency to the apps that need the secret? for backwards compatibility, otherwise this is a breaking change

Reply:
and also to mnp

Reply:
Yes this can break applications do not declare the dependency, that's why you see the tag alert:deployment. At the same time, an application that uses the keycloak admin api in its logic has always been intended to declare the hard dependency to accounts. Now by chance other applications could include account and everything works, but that's just why the odds are good this is true that things usually work regardless the undeclared dependency.
That's probably just a semantic detail, but I'd rather use the breaking-change tag for real changes in the api or more in general the way things are intended to be.

Reply:
agree